modules/impact.base.coll-entry.d.ts

```diff
@@ -2,14 +2,43 @@ export {};
 
 declare global {
   namespace ig {
+    enum COLL_SHADOW_TYPE {
+      DEFAULT = 0,
+      STATIC_SIZE = 1,
+      RECTANGULAR = 2, 
+    }
+    namespace CollEntry {
+      interface Shadow {
+        type: ig.COLL_SHADOW_TYPE;
+        size: number;
+        scaleY: number;
+        offset: Vec2;
+      }
+    }
     interface CollEntry extends ig.Class {
       pos: Vec3;
+      vel: Vec3;
       size: Vec3;
       relativeVel: Vec2;
+      type: COLLTYPE;
+      weight: number;
+      zGravityFactor: number;
+      baseZPos: number;
+      zBounciness: number;
+      level: number;
+      shadow: CollEntry.Shadow;
```

Comment:
Misordered properties, and while it's not from your changes, `relativeVel` is wrongly typed.
```suggestion
      type: COLLTYPE;
      size: Vec3;
      weight: number;
      relativeVel: number;
      zBounciness: number;
      zGravityFactor: number;
      pos: Vec3;
      level: number;
      baseZPos: number;
      shadow: CollEntry.Shadow;
      vel: Vec3;
```